src/components/organisms/contact-content/contact-content.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from "react";
+import { contactImg } from "images/Index";
+import DefaultImage from "components/atoms/image/image";
+import ContactHeading from "components/molecules/contact-heading/contact-heading";
+import ContactForm from "components/molecules/contact-form/contact-form";
+import React from "react";
+
+const ContactContent = () => {
+
+  return ( 
+    <div className="flex w-full ">
+      <div className="lg:w-[45%] hidden xl:flex lg:flex justify-center items-center">
+
+        <DefaultImage 
+          src={"https://res.cloudinary.com/phantom1245/image/upload/v1675875866/techishiring/techish_qcfih9.png" || contactImg}
+          alt="contact-img" 
+          w={"100%"}
+          h={"auto"}
+        />
```

Comment:
```suggestion        
        <Image src={*Image*}  alt="Contact Form Image" {/* *other necessary attributes* */}  />
```

This would be an example of how to use next/image.